src/view/components/DataTable/index.tsx

```diff
@@ -10,6 +10,7 @@ interface OwnPropsType {
   headers: string[];
   values: any[];
   routed?: RoutedPropsType;
+  onEntryPress(uuid: string): any;

```

Comment:
I don't know the return type of an onPress

Reply:
I tried void typescript didn't like it one bit but Ill rename

Reply:
Of course it didn't.